Cargo.toml

```diff
@@ -15,9 +15,6 @@ version = "0.5.0"
 features = ["stm32f303xc", "rt", "stm32-usbd"]
 targets = ["thumbv7em-none-eabihf"]
 
-[badges]
-travis-ci = { repository = "stm32-rs/stm32f3xx-hal" }
-
```

Comment:
Is there no replacement for GHA?

Reply:
There is not, but it doesn't matter. The badges have been deprecated and are no longer shown on crates.io anyway.
https://github.com/rust-lang/crates.io/issues/2436